hardhat.config.ts

```diff
@@ -6,6 +6,7 @@ import "@nomiclabs/hardhat-waffle";
 import "@typechain/hardhat";
 import "hardhat-gas-reporter";
 import "solidity-coverage";
+import { networks } from "./hardhat.network";
```

Comment:
What is the benefit of moving this to a separate file?

Reply:
I followed what [this repo](https://github.com/IPOR-Labs/ipor-protocol/blob/64f90ca96bd9b99036cf6d5e18a136f73f191d9d/hardhat.network.ts) did and noticed that they had it in a separate file.